orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/metrics/AtlasQueueMonitorTest.kt

```diff
@@ -55,14 +65,15 @@ object AtlasQueueMonitorTest : SubjectSpek<AtlasQueueMonitor>({
     on { counter(eq("queue.duplicate.messages"), anyVararg<String>()) } doReturn duplicateCounter
     on { counter("queue.lock.failed") } doReturn lockFailedCounter
     on { timer("queue.message.lag") } doReturn messageLagTimer
+    on { counter(eq("zombie.execution"), any<Iterable<Tag>>()) } doReturn zombieCounter
```

Comment:
`queue.zombies` ? 

`queue.zombie.executions`?

Figure keeping the tag names somewhat consistent for queues might prove beneficial